core/schema/container.go

```diff
@@ -399,7 +399,8 @@ func (s *containerSchema) Install() {
 				`- For setting the EXPOSE OCI field when publishing the container`).
 			ArgDoc("port", `Port number to expose`).
 			ArgDoc("protocol", `Transport layer network protocol`).
-			ArgDoc("description", `Optional port description`),
+			ArgDoc("description", `Optional port description`).
+			ArgDoc("experimentalSkipHealthCheck", `Skip the health check when run as a service.`),
```

Comment:
I feel like I'm missing something super simple here... 

If I comment out the ```ArgDoc("experimentalSkipHealthCheck", `Skip the health check when run as a service.`),``` line the SDKs generate & tests pass as expected, But with it seems like the GraphQL server can't startup due to the following error:

```
Error: input:1: pipeline.pipeline.container.from.withExec.withEnvVariable.withExec.withWorkdir.withDirectory.withMountedCache.withExec.withMountedDirectory.withMountedCache.withExec.file process "go run ./cmd/introspect" did not complete successfully: exit code: 1

Stdout:

Stderr:
panic: field withExposedPort has no such argument: "experimentalSkipHealthCheck"

goroutine 1 [running, locked to thread]:
github.com/dagger/dagger/dagql.Field[...].ArgDoc(_, {_, _}, {_, _, _})
        /app/dagql/objects.go:551 +0x274
github.com/dagger/dagger/core/schema.(*containerSchema).Install(0xc000070500)
        /app/core/schema/container.go:403 +0x7e5d
github.com/dagger/dagger/core/schema.(*CoreMod).Install(0x15014a0?, {0xc0000d61e0?, 0x40fade?}, 0xc0000a3b30)
        /app/core/schema/coremod.go:49 +0x4a2
github.com/dagger/dagger/core/schema.New({0x1508a00, 0x1e3b0c0}, {0x0, {{0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0}, {0x0, ...}, ...}, ...})
        /app/core/schema/schema.go:65 +0x3f3
main.main()
        /app/cmd/introspect/main.go:12 +0x4e
exit status 2
exit status 1
```

Reply:
Looks like the field is `ExperimentalSkipHealthcheck` - the `c` in `check` needs to be capitalized for it to match (or vice versa)

Reply:
`"experimentalSkipHealthCheck"` vs `"experimentalSkipHealthcheck"` - Capitalized `C`.